bin/lando/setup-multisite.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Check if this is already a multisite installation
+wp core is-installed --network
+[ $? -eq 0 ] && echo "WARNING: Multisite already setup, no need to do anything" && exit 1
+
+echo "Converting installation to multisite..."
+wp core multisite-convert --path=$LANDO_WEBROOT --title="VIP Go Dev Network" --subdomains
```

Comment:
Does this automatically add the necessary constants to `wp-config`?

Reply:
Looks like it does: `and adds the multisite constants to wp-config.php` (https://developer.wordpress.org/cli/commands/core/multisite-convert/)

Reply:
Yep, it actually tells you it's applying the wp-config.php changes when you run the command :)